src/loginServices.js

```diff
@@ -752,6 +767,7 @@ export async function logout(okapiUrl, store, queryClient) {
     // clear shared storage
     .then(localforage.removeItem(SESSION_NAME))
     .then(localforage.removeItem('loginResponse'))
+    .then(() => localStorage.removeItem(TENANT_LOCAL_STORAGE_KEY))
```

Comment:
Can we move this above with the other `localStorage.removeItem()` entries? If not, please add a comment explaining why it needs to be handled separately.